docs/apis/user.yaml

```diff
@@ -18,56 +18,86 @@ servers:
 tags:
   - name: users
     description: Operations related to user management
+  - name: users-by-path
+    description: Operations related to user management using hierarchical handle paths
+
 paths:
```

Comment:
Should we allow updating a user's organization unit with PUT request? Current example denotes updating the ou

Reply:
Yes.. Its possible to move the user from one OU to another.. Is there any issue you see with that?